packages/components/src/templates/next/components/complex/Contentpic/Contentpic.tsx

```diff
@@ -0,0 +1,39 @@
+import type { VariantProps } from "tailwind-variants"
+
+import type { ContentpicProps as BaseContentpicProps } from "~/interfaces"
+import { tv } from "~/lib/tv"
+import { Prose } from "../../native"
+
+const contentpicStyles = tv({
+  slots: {
+    container: "flex flex-col gap-7 py-7 md:flex-row",
+    image: "inset-0 h-full w-full object-cover lg:absolute",
+    imageContainer:
+      "relative max-h-[400px] w-full md:h-[240px] md:max-h-full md:w-[200px]",
```

Comment:
issue: there is a range of viewports that has the image covering the prose content.

https://github.com/user-attachments/assets/91331470-839c-4d75-b84e-3b59e687d836

Actually curious why do we need relative and absolute here? Seems like it should be possible for the image and the content to wrap around themselves normally.

Reply:
was lazy and reused styling from infopic. That one needed because unknown width. Now that this is constrained I think don't need liow. Let me check.